tests/migration_test.go

```diff
@@ -1504,84 +1502,32 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
-				dv = tests.NewRandomDataVolumeWithRegistryImport(url, util.NamespaceTestDefault, k8sv1.ReadWriteOnce)
+				dv = tests.NewRandomDataVolumeWithRegistryImport(url, util.NamespaceTestDefault, k8sv1.ReadWriteMany)
 				dv.Spec.PVC.Resources.Requests["storage"] = quantity
 				_, err = virtClient.CdiClient().CdiV1beta1().DataVolumes(dv.Namespace).Create(context.Background(), dv, metav1.CreateOptions{})
 				Expect(err).ToNot(HaveOccurred())
-
-				wffcPod = tests.RenderPod("wffc-temp-pod", []string{"echo"}, []string{"done"})
-				wffcPod.Spec.Containers[0].VolumeMounts = []k8sv1.VolumeMount{
-
-					{
-						Name:      "tmp-data",
-						MountPath: "/data/tmp-data",
-					},
-				}
-				wffcPod.Spec.Volumes = []k8sv1.Volume{
-					{
-						Name: "tmp-data",
-						VolumeSource: k8sv1.VolumeSource{
-							PersistentVolumeClaim: &k8sv1.PersistentVolumeClaimVolumeSource{
-								ClaimName: dv.Name,
-							},
-						},
-					},
-				}
-
-				By("pinning the wffc dv")
-				wffcPod, err = virtClient.CoreV1().Pods(util.NamespaceTestDefault).Create(context.Background(), wffcPod, metav1.CreateOptions{})
-				Expect(err).ToNot(HaveOccurred())
-				Eventually(ThisPod(wffcPod), 120).Should(BeInPhase(k8sv1.PodSucceeded))
-
-				By("waiting for the dv import to pvc to finish")
-				Eventually(ThisDV(dv), 600).Should(HaveSucceeded())
-
-				// Prepare a NFS backed PV
-				By("Starting an NFS POD to serve the PVC contents")
-				nfsPod := storageframework.RenderNFSServerWithPVC("nfsserver", dv.Name)
-				nfsPod, err = virtClient.CoreV1().Pods(util.NamespaceTestDefault).Create(context.Background(), nfsPod, metav1.CreateOptions{})
-				Expect(err).ToNot(HaveOccurred())
-				Eventually(ThisPod(nfsPod), 120).Should(BeInPhase(k8sv1.PodRunning))
-				nfsPod, err = ThisPod(nfsPod)()
-				Expect(err).ToNot(HaveOccurred())
-				nfsIP := libnet.GetPodIpByFamily(nfsPod, k8sv1.IPv4Protocol)
-				Expect(nfsIP).NotTo(BeEmpty())
-				// create a new PV and PVC (PVs can't be reused)
-				By("create a new NFS PV and PVC")
-				os := string(cd.ContainerDiskFedoraTestTooling)
-				tests.CreateNFSPvAndPvc(pvName, util.NamespaceTestDefault, "5Gi", nfsIP, os)
+				pvName = dv.Name
 			})
 
 			AfterEach(func() {
-				By("Deleting NFS pod")
-				// PVs can't be reused
-				tests.DeletePvAndPvc(pvName)
-
 				if dv != nil {
 					By("Deleting the DataVolume")
 					Expect(virtClient.CdiClient().CdiV1beta1().DataVolumes(dv.Namespace).Delete(context.Background(), dv.Name, metav1.DeleteOptions{})).To(Succeed())
 					dv = nil
 				}
-				if wffcPod != nil {
-					By("Deleting the wffc pod")
-					err = virtClient.CoreV1().Pods(util.NamespaceTestDefault).Delete(context.Background(), wffcPod.Name, metav1.DeleteOptions{})
-					Expect(err).ToNot(HaveOccurred())
-					wffcPod = nil
-				}
 			})
 
 			It("[test_id:2653] should be migrated successfully, using guest agent on VM with default migration configuration", func() {
```

Comment:
/hold

I don't see this test executed, both the 1.23 sig compute lane run and the 1.23 sig compute migrations show this test as skipped, same for the one below. I'd expect these tests to get executed in either lane.

I suspect that this test is skipped now due to missing test expectations. @akalenyu, can you please take a look why the tests are getting skipped?

Reply:
Thanks for catching this. discussing reasoning & ways forward [above](https://github.com/kubevirt/kubevirt/pull/7324#discussion_r830621517)